content/docs/concepts/metric_types.md

```diff
@@ -48,6 +48,10 @@ A _histogram_ samples observations (usually things like request durations or
 response sizes) and counts them in configurable buckets. It also provides a sum
 of all observed values.
 
+Since each observation bucket holds the total count of observations less than or
```

Comment:
Yes, let's improve the wording and emphasis, and linking the term in wikipedia is also good.

However, we try to avoid justifications for technical decisions and design in the core documentation as that will blow them up immensely. This is more something we would have in a blog post, or perhaps in one of the less terse parts of the docs like the best-practice pages (it might have a place in https://prometheus.io/docs/practices/histograms/ ).